packages/flutter_tools/lib/src/artifacts.dart

```diff
@@ -299,3 +299,89 @@ class LocalEngineArtifacts extends Artifacts {
     throw Exception('Unsupported platform $platform.');
   }
 }
+
+/// A fuchsia host specific implementation of [Artifacts]
+///
+/// Certain binaries will have different locations which are not relative to
+/// a flutter tool directory.
+class FuchsiaHostArtifacts extends CachedArtifacts {
+  /// Create a [FuchsiaHostArtifacts] with the fuchsia host tools in `_fuchsiaHostTools`.
```

Comment:
Since we're making the member variable public (see comment below), you can use a dartdoc link instead of backticks.